test/y2storage/boot_requirements_errors_test.rb

```diff
@@ -472,39 +406,30 @@
     context "in a S/390 system" do
       let(:architecture) { :s390 }
       let(:efiboot) { false }
-      let(:use_lvm) { false }
-      let(:partitions) { [zipl_partition] }
-
-      before do
-        allow(dev_sda).to receive(:is?).with(:dasd).and_return(dasd)
-        allow(dev_sda).to receive(:type).and_return(type)
-        allow(dev_sda).to receive(:format).and_return(format)
-      end
-
-      let(:dasd) { false }
-      let(:type) { Y2Storage::DasdType::UNKNOWN }
-      let(:format) { Y2Storage::DasdFormat::NONE }
 
       context "when there is no root" do
-        let(:root_filesystem) { nil }
+        let(:scenario) { "false-swaps" }
+
         include_examples "unknown boot disk"
       end
 
-      context "using a zfcp disk as boot disk" do
+      # TODO: find coresponding xml files for it
+      # and ideally from real usage
+      xcontext "using a zfcp disk as boot disk" do
```

Comment:
Probably, in the scenario `kinds-of-dasd.xml`, we could add a new DASD with `<transport>FCoE</transport>`. This should be enough.

Reply:
ok, will try.

Reply:
Anyway, I see we have support for Dasd in [fake_device_factory.rb](https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/fake_device_factory.rb#L213). It should be easy to create an scenario with Dasd (even with a specific type and format).